docs/store-operations/tax/tax-properties.mdx

```diff
@@ -1,25 +1,27 @@
 ---
-title: Tax Properties for Products
-keywords: get all products, tax, tax properties, 
+title: Tax Properties
+keywords: tax, tax properties,
 ---
 
-# Tax Properties for Products
+# Tax Properties
 
-The Tax Properties API allows merchants to vary the tax information they send to tax providers for tax calculation. In some jurisdictions, tax rates for products vary by product composition. For example, accurate taxation of alcohol may require information about alcohol percentage. Using tax properties, providers can return more accurate tax quotes. The examples in this guide expand on this use case. 
+The Tax Properties API allows merchants to vary the tax information they send to tax providers for tax calculation. In some jurisdictions, tax rates for products vary by product composition. For example, accurate taxation of alcohol may require information about alcohol percentage. Using tax properties, providers can return more accurate tax quotes. The examples in this guide expand on this use case.
 
 Tax providers may sometimes need multiple inputs that can vary between different types of product. For example, for products with alcohol, the alcohol percentage and the volume sold affect alcohol taxes. As shown in this guide, merchants can provide both pieces of information to tax providers.
```

Comment:
As above, I think we can drop this paragraph - it's largely a repeat and seems to only be pointing out that users can have multiple product tax properties.